src/Containers/Microsoft.NET.Build.Containers/ContainerBuilder.cs

```diff
@@ -2,11 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Microsoft.NET.Build.Containers.Resources;
+using System.Text;
 
 namespace Microsoft.NET.Build.Containers;
 
 public static class ContainerBuilder
 {
+
+    private static void WriteLogMessage(LogMessage message) {
+        if (message.level is ProgressMessageLevel.Info) {
+            var builder = new StringBuilder("Containerize: ");
+            builder.AppendFormat(message.messageFormat, message.formatArgs);
+            Console.WriteLine(builder.ToString());
+        }
+        // TODO: dropping Trace level messages - this should be ok for the net472 VS experience. CI systems will get them in binlogs.
```

Comment:
minor: trace level should be managed globally via configuration.

Should we adopt this common scheme?
https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line